node/chainSimulator/components/coreComponents.go

```diff
@@ -89,9 +90,13 @@ type ArgsCoreComponentsHolder struct {
 	NumShards           uint32
 	WorkingDir          string
 
-	MinNodesPerShard  uint32
-	MinNodesMeta      uint32
-	RoundDurationInMs uint64
+	MinNodesPerShard            uint32
+	MinNodesMeta                uint32
+	RoundDurationInMs           uint64
+	ConsensusGroupSize          uint32
+	MetaChainConsensusGroupSize uint32
+	CreateGenesisNodesSetup     func(nodesFilePath string, addressPubkeyConverter core.PubkeyConverter, validatorPubkeyConverter core.PubkeyConverter, genesisMaxNumShards uint32) (sharding.GenesisNodesSetupHandler, error)
+	CreateRatingsData           func(arg rating.RatingsDataArg) (process.RatingsInfoHandler, error)
```

Comment:
Why funcs here? These should be inside runType

Reply:
Can't put them in runtype because runtype uses core components

Reply:
`MX-12902` Task for RunTypeCoreComponents